clients/admin-ui/src/pages/management/consent.tsx

```diff
@@ -0,0 +1,508 @@
+              ({
+                purpose: po.purpose,
+                is_included: po.is_included,
+                is_consent:
+                  po.required_legal_basis === TCFLegalBasisEnum.CONSENT,
+                is_legitimate_interest:
+                  po.required_legal_basis ===
+                  TCFLegalBasisEnum.LEGITIMATE_INTERESTS,
+              } as FormPurposeOverride)
+          )
+        : [],
+    }),
+    [tcfPurposeOverrides]
+  );
+
+  return (
+    <Layout title="Consent Configuration">
+      {isHealthCheckLoading ||
+      isPurposesLoading ||
+      isTcfPurposeOverridesLoading ||
+      isApiConfigSetLoading ||
+      isConfigSetLoading ? (
+        <Flex justifyContent="center" alignItems="center" height="100%">
+          <Spinner />
+        </Flex>
+      ) : (
+        <Box data-testid="consent-configuration">
+          <Heading marginBottom={4} fontSize="2xl">
+            Consent settings
+          </Heading>
+          <Box>
+            <Box backgroundColor="gray.50" borderRadius="4px" padding="14px">
+              <Text
+                fontSize="md"
+                fontWeight="bold"
+                lineHeight={5}
+                color="gray.700"
+              >
+                Transparency & Consent Framework settings
+              </Text>
+              <Text
+                mb={2}
+                mt={3}
+                fontSize="sm"
+                lineHeight="5"
+                fontWeight="medium"
+                color="gray.700"
+              >
+                TCF status:{" "}
+                {isTcfEnabled ? (
+                  <Badge backgroundColor="green.100">Enabled </Badge>
+                ) : (
+                  <Badge backgroundColor="red.100">Disabled</Badge>
+                )}
+              </Text>
+              <Text
+                fontSize="sm"
+                lineHeight="5"
+                fontWeight="medium"
+                color="gray.700"
+              >
+                To {isTcfEnabled ? "disable" : "enable"} TCF, please contact
+                your Fides Administrator or{" "}
+                <DocsLink href="https://ethyca.com">Ethyca support</DocsLink>
+              </Text>
+            </Box>
+
+            <Box
+              mt="24px"
+              backgroundColor="gray.50"
+              borderRadius="4px"
+              padding="14px"
+            >
+              <Text
+                fontSize="md"
+                fontWeight="bold"
+                lineHeight={5}
+                color="gray.700"
+                mb={3}
+              >
+                Vendor overrides
+              </Text>
+              {isTcfEnabled ? (
+                <>
+                  <Text
+                    marginBottom={2}
+                    fontSize="sm"
+                    lineHeight="5"
+                    fontWeight="medium"
+                    color="gray.700"
+                  >
+                    Override vendor purposes:{" "}
```

Comment:
The designs have a tooltip, do we know what should go there?

Reply:
I don't but I'll ask Michael

Reply:
There is now a tooltip using text provided by Michael